differs/metadata_diff.go

```diff
@@ -53,14 +59,50 @@ func (a MetadataAnalyzer) Analyze(image pkgutil.Image) (util.Result, error) {
 }
 
 func getMetadataDiff(image1, image2 pkgutil.Image) (MetadataDiff, error) {
-	m1 := getMetadataList(image1)
-	m2 := getMetadataList(image2)
+	m1, err := getMetadataList(image1)
+	if err != nil {
+		return MetadataDiff{}, err
+	}
+	m2, err := getMetadataList(image2)
+	if err != nil {
+		return MetadataDiff{}, err
+	}
 
 	adds := util.GetAdditions(m1, m2)
 	dels := util.GetDeletions(m1, m2)
 	return MetadataDiff{adds, dels}, nil
 }
 
-func getMetadataList(image pkgutil.Image) []string {
-	return image.Config.Config.AsList()
+func getMetadataList(image pkgutil.Image) ([]string, error) {
+	configFile, err := image.Image.ConfigFile()
+	if err != nil {
+		return nil, err
+	}
+	c := configFile.Config
+	return []string{
+		fmt.Sprintf("Hostname: %s", c.Hostname),
```

Comment:
Do these need newlines?

Reply:
they don't, this is a list of strings that gets thrown into a generic result struct and outputted separately